server/infrastructure/saved_players_repository.py

```
@@ -0,0 +1,124 @@
+                )
+            
+            return SavedPlayer(**player_doc.to_dict())
+        except HTTPException:
+            raise
+        except Exception as e:
+            raise HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=f"Failed to retrieve player: {str(e)}"
+            )
+    
+    async def delete_player(self, user_id: str, player_id: str) -> DeletePlayerResponse:
```

Ruff (0.14.1)

Unused method argument: `user_id`

(ARG002)